src/useTock.ts

```diff
@@ -268,12 +270,14 @@ const useTock: (
       startLoading();
       const body = payload
         ? {
-            payload: payload,
-            userId: userId,
+            payload,
+            userId,
+            locale
```

Comment:
After verification, Jackson will indeed take a string matching an ISO 639 code. It may be worth noting in the documentation, though it could be considered prerequisite knowledge as it is the standard for encoding locales.